src/main/java/com/flansmod/common/guns/ItemGun.java

```diff
@@ -1286,9 +1295,9 @@ public boolean reload(ItemStack gunStack, GunType gunType, World world, Entity e
                         int bulletsInThisSlot = item.getMaxDamage() - item.getItemDamage();
                         boolean isPreferred = ((ItemShootable) item.getItem()).type.shortName.equals(preferredAmmoShortname);
                         if (isPreferred) {
-                            if ((bestSlotIsPreferred && bulletsInThisSlot > bulletsInBestSlot) || !bestSlotIsPreferred) {
+                            if (!bestSlotIsPreferred || bulletsInThisSlot > bulletsInBestSlot) {
                                 bestSlot = j;
-				bulletsInBestSlot = bulletsInThisSlot;
+				                bulletsInBestSlot = bulletsInThisSlot;
```

Comment:
Appreciate the formatting, but what's with this tab here? Or is that just gh being weird.
Doesn't matter too much eithr way

Reply:
Yeah not sure, it was messed up in my IDE and it now looks correct but gh doesnt seem to agree. 
<img width="420" alt="image" src="https://user-images.githubusercontent.com/41086203/212501420-2abd34f1-f550-43a3-8778-3d2240d3e3ac.png">